### PR TITLE
Add missing copyright to non generated files

### PR DESCRIPTION
**Pull request by MStoykov**

### CODE REVIEW

File: js/empty_iteartions_bench_test.go [hunk 1]
```diff
@@ -1,3 +1,23 @@
+/*
```
Comment (imiric) [inline js/empty_iteartions_bench_test.go:1]: Just noticed there's a typo in this filename: `iteartions`.

**Review by imiric — APPROVED** [review]
Might as well fix the filename typo here?

**Review by imiric — APPROVED** [review]

**Review by codebien — APPROVED** [review]
LGTM

**Review by yorugac — COMMENTED** [review]
There are also js/tc39 tests without copyright header too, though they have the note of "Heavily influenced by the fantastic work by @dop251 for https://github.com/dop251/goja". Not sure how much "heavily" is here: are they considered goja's?

**Comment by MStoykov** [comment]
the `(no)?race` ones are direct copies while the other one is still probably 50%+ just the goja file (especially if you remove the big arrays of specific tests or types of tests we do or do *not* run. Given that I really don't feel we can add our copyright.

**Pull request merged**